marlowe-dashboard-client/src/Play/Types.purs

```diff
@@ -68,8 +69,9 @@ instance actionIsEvent :: IsEvent Action where
   toEvent (AddNewWallet _) = Just $ defaultEvent "AddNewWallet"
   toEvent ToggleMenu = Just $ defaultEvent "ToggleMenu"
   toEvent (SetScreen _) = Just $ defaultEvent "SetScreen"
-  toEvent (SetCard _) = Just $ defaultEvent "SetCard"
-  toEvent (ToggleCard _) = Just $ defaultEvent "ToggleCard"
+  toEvent (OpenCard _) = Nothing
+  toEvent (ToggleCard _) = Nothing
```

Comment:
In the description you said you removed the `ToggleCard`

Reply:
oops, I missed this - thanks